src/main/scala/chisel3/stage/ChiselAnnotations.scala

```diff
@@ -46,8 +46,9 @@ case class ChiselGeneratorAnnotation(gen: () => RawModule) extends NoTargetAnnot
 
   /** Run elaboration on the Chisel module generator function stored by this [[firrtl.annotations.Annotation]]
     */
-  def elaborate: ChiselCircuitAnnotation = try {
-    ChiselCircuitAnnotation(Builder.build(Module(gen())))
+  def elaborate: AnnotationSeq  = try {
```

Comment:
This breaks testers2 which expects to be able to reference a circuit:
```
[info] Compiling 29 Scala sources to .../testers2/target/scala-2.11/classes ...
[error] /vm/home/jenkins/workspace/chisel3-pl_2/testers2/src/main/scala/chisel3/tester/backends/treadle/TreadleExecutive.scala:30:49: value circuit is not a member of firrtl.AnnotationSeq
[error]     val circuit = generatorAnnotation.elaborate.circuit
[error]                                                 ^
```

Reply:
Since we're still pre-3.2, is it safe to fix this on the testers2 side? Something like:
```scala
val circuit = generatorAnnotation.elaborate._1.circuit
```